### DISCUSSION TITLE
Question for Hamd Waseem.

### DISCUSSION THREAD

**Discussion by MKM12345** [hamdivazim/CodeGlob#3] [General] [answered]
Ok @hamdivazim, just want to know, if you implement [my idea](https://github.com/hamdivazim/CodeGlob/discussions/2) that I posted, you'll need a database. My question is can you create a large enough database for Code Glob and can it include more features like login, signup, reset password, change profile picture and settings etc? And can you complete in the given time you have, as I understand you have other _commitments_ to complete.

**Comment by hamdivazim-archive** [comment, marked as answer]
Yes, we are planning to use django. Django comes with an authentication app out of the box, and it also comes with SQL support, which is a powerful database and should be relatively scalable. So in short, django comes with all those features out of the box, which is why we chose to develop backend using this over something like PHP.
